File: AbstractFactoryDesignPattern/NotificationService_Example/Sender/SMSSendingService.java
package AbstractFactoryDesignPattern.NotificationService_Example.Sender;

import AbstractFactoryDesignPattern.NotificationService_Example.Message.Message;

public class SMSSendingService implements Sender {
	@Override
	public void sendMessage(Message message) {
		System.out.println("Sending Message via carrir api(): " + message.formatMessage());
	}
}

File: AbstractFactoryDesignPattern/NotificationService_Example_Python/SenderService/SMSSendingService.py
from AbstractFactoryDesignPattern.NotificationService_Example_Python.Message.NotificationMessage import \
	NotificationMessage
from AbstractFactoryDesignPattern.NotificationService_Example_Python.SenderService.NotificationService import \
	NotificationService


class SMSSendingService(NotificationService):

	def sendMessage(self, message: NotificationMessage) -> None:
		print(f"Sending Notification Message VIA Carrier channels: {message.formatMessage()}")
